engine/installation/windows/docker-ee.md

```diff
@@ -32,14 +32,14 @@ full list of prerequisites.
 
 1.  Open a PowerShell command prompt, and type the following commands.
 
-    ```ps
+```PowerShell
```

Comment:
The indentation is on purpose, and has to be there because these code examples are in the numbered list; removing the whitespace results in the list being terminated, and a new list started; here's the before and after:

<img width="909" alt="screen shot 2017-09-26 at 10 06 47" src="https://user-images.githubusercontent.com/1804568/30849462-b5506bd8-a2a2-11e7-9dda-f574fea759dc.png">

After:

<img width="946" alt="screen shot 2017-09-26 at 10 07 16" src="https://user-images.githubusercontent.com/1804568/30849468-bcb2cfc4-a2a2-11e7-9d51-ec0b2a52b679.png">


Notice that the numbering starts at `1.` again